sdk/openai/Azure.OpenAI.Inference/tests/Samples/Readme.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
nit:  Filename should be README.md for consistency

Reply:
this is the cs file, the md is named README.md https://github.com/Azure/azure-sdk-for-net/pull/33457/files#diff-f4ce28190a20fc7c68498a6d365edf0090f20d7215c9833d94562feaa8292490

Reply:
... yeah.   uh, reading is hard.  

<img src="https://media3.giphy.com/media/yvBAuESRTsETqNFlEl/giphy.gif"/>